entries/s/subito.it.json

```
@@ -0,0 +1,15 @@
+{
+  "Subito": {
+    "domain": "subito.it",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://assistenza.subito.it/hc/articles/11599185074077",
```

Using my very limited Italian, it sounds like the page describes what to do if the code isn't received. If so, it would probably be better to use the recovery attribute.
```suggestion
    "recovery": "https://assistenza.subito.it/hc/articles/11599185074077",
```

---

Yes, you're right! I thought it belonged to documentation by default. Thank you for updating!